### PR TITLE
Add `zowe config` group examples in Help

### PR DESCRIPTION
**Pull request by namanpatel1112**
Resolves https://github.com/zowe/zowe-cli/issues/1061

### CODE REVIEW

**Comment by namanpatel1112** [comment]
I added examples for the `zowe config` group of commands. But, I cannot see the details about the failing jenkins check.

**Comment by t1m0thyj** [comment]
> I added examples for the `zowe config` group of commands. But, I cannot see the details about the failing jenkins check.

The changelog check is failing. Not sure why the GH workflow didn't run though, it should also have detected the missing changelog 🤔

**Comment by awharn** [comment]
> 
> 
> > I added examples for the `zowe config` group of commands. But, I cannot see the details about the failing jenkins check.
> 
> The changelog check is failing. Not sure why the GH workflow didn't run though, it should also have detected the missing changelog 🤔

It doesn't seem to be configured to run against PRs that target `next`, just `master`.

**Review by awharn — APPROVED** [review]

File: packages/imperative/src/config/cmd/list/list.definition.ts [hunk 1]
```diff
@@ -39,5 +39,27 @@ export const listDefinition: ICommandDefinition = {
                 "For example, specify in addition to '--locations' to get a list of config file paths only.",
             type: "boolean"
         }
+    ],
+    examples: [
+        {
+            description: "List property names for a specified config property.",
+            options: `"default"`
```
Comment (t1m0thyj) [inline packages/imperative/src/config/cmd/list/list.definition.ts:46]: Should this be plural "defaults" to match the property name in zowe.config.json?

File: packages/imperative/src/config/cmd/list/list.definition.ts [hunk 2]
```diff
@@ -39,5 +39,27 @@ export const listDefinition: ICommandDefinition = {
...
+        },
+        {
+            description: "List only root level property names for a specified config property.",
+            options: `"default" --root`
```
Comment (t1m0thyj) [inline packages/imperative/src/config/cmd/list/list.definition.ts:50]: Same question as above, should the property name be plural?

File: packages/imperative/src/config/cmd/list/list.definition.ts (hunk 1 above)
Comment (namanpatel1112) [reply packages/imperative/src/config/cmd/list/list.definition.ts:46]: Right, I'll make it plural.

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**